test/query.test.js

```diff
@@ -222,6 +222,7 @@ describe('query', () => {
         new values.Query({
           lambda: '_',
           expr: { let: { x: 1, y: 2 }, in: [{ var: 'x' }, { var: 'y' }] },
+          api_version: '3',
```

Comment:
Without the new `api_version` field here, this test was failing

Reply:
I was expecting a separate test specifically for this, similar to `legacy queries/lambdas have default api_version` where you call the `Query()` function and assert the returned value contains the `api_version` equals the latest api version driver is using, this was an accidental coincidence